components/builder-worker/src/runner/mod.rs

```diff
@@ -399,6 +404,23 @@ impl Runner {
     async fn fetch_origin_secret_key(
         &self)
         -> std::result::Result<std::path::PathBuf, builder_core::Error> {
+        if env::var_os("HAB_FUNC_TEST").is_some() {
+            let test_key = "bobo".to_string();
+            let res =
+                self.depot_cli
+                    .fetch_origin_secret_key(self.job().origin(),
+                                             &test_key,
+                                             self.workspace.key_path())
+                    .await;
+            if res.is_err() {
+                debug!("Failed to fetch origin secret key, err={:?}, path={:?}",
+                       res,
+                       self.workspace.key_path());
+            };
+
+            return res;
+        }
+
```

Comment:
I think you can avoid all of this repetitive code by creating a `token` variable and populating it with either bobo or the `bldr_token` depending on `HAB_FUNC_TEST`

Reply:
updated